packages/mesos-modules/build

```diff
@@ -1,15 +1,53 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
+# Update the image and download build tools.
+apt-get update -qq
+apt-get install -y autoconf libtool make pkg-config
```

Comment:
the openssl, curl, python we bundle in DC/OS, shouldn't use the ones from the base image docker.